### ISSUE TITLE
Loss is becoming more negative, correct?

### ISSUE THREAD

**Issue by tessavdheiden** [vvanirudh/srnn-pytorch#7]
Hi guys,

Does anybody know why the loss is becoming more negative and if this is a good thing? My understanding is, that the loss should get closer to 0.

Best
Tessa

**Comment by WERush** [comment]
The loss is the target entropy. Hence, negative value is normal.

**Comment by tessavdheiden** [comment]
@WERush Please look at the paper, it says the negative log-likelihood is minimized, which is negative if the input is bigger than 1, which normally cannot be true, since a probability should be less than 1!!!

**Comment by WERush** [comment]
@tessavdheiden  Please check the paper again. This is the entropy minimization and the input is not the probability. You can find the specific implementation in this code https://github.com/WERush/superSRNN/blob/new_v3/srnn/criterion.py

**Comment by WERush** [comment]
@tessavdheiden  Please check the paper again. The input is not a probability, and you can find the implementation in this code https://github.com/vvanirudh/srnn-pytorch/blob/930e88705e8f59f97dead8f2a28cdea89b1220d3/srnn/criterion.py

**Comment by tessavdheiden** [comment]
The input of the log is the probability of the predicted Gaussian distribution (suppose values between -1 and 1), or?

**Comment by NankyLiu** [comment]
I just run the raw sample.py  in my windows,found that there was no dataset ewap_dataset/seq_eth、ucy_crowd or KITTITrackingData/GT  mentioned at line 28;and I noticed in the issues that we need to check homography matrix when train and test; so
Could you  please share with me the whole dataset (image files) + homography matrix ?
or share URL for each of them?and it was the first time I run this code ,could you please give me any other  suggestions need to be noticed? 

Thank you very much
